src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -114,6 +114,9 @@
     <AndroidResource Include="Resources\**\*.axml" />
     <AndroidResource Include="Resources\**\*.xml" />
     <AndroidResource Include="Resources\**\*.png" />
+
+    <PackageReference Include="Xamarin.GooglePlayServices.Auth" Version="119.0.0">
```

Comment:
CC @jfversluis @brminnick @pictos Do you folks think we can add this dependency?

Reply:
Lets solve this before I address the other issues you found, thanks for the review!

Reply:
No, thank you @krdmllr for this awesome PR :)
I am not sure if we can accept it since it brings some dependency, but this feature looks super useful!